jetson/percep/camera.cpp

```diff
@@ -333,7 +353,7 @@ void Camera::record_ar_init() {
 
   if(vidWrite.isOpened() == false)
   {
-	  cerr << "ar record didn't open\n";
+	    cout << "ar record didn't open\n";
```

Comment:
Is there an extra space before this cout?

Reply:
Fixed